backend-beeyond/src/main/java/at/htl/beeyond/validation/NamespaceValid.java

```diff
@@ -0,0 +1,18 @@
+package at.htl.beeyond.validation;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.*;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = NamespaceValidValidator.class)
+@Documented
+public @interface NamespaceValid {
+
+    String message() default "{at.htl.beeyond.validation.Namespace.message}";
```

Comment:
`{at.htl.beeyond.validation.Namespace.message}` -> `{at.htl.beeyond.validation.NamespaceValid.message}`. Use the name of the annotation.